tests/test_patch.py

```diff
@@ -54,23 +52,6 @@ def test_distributed():
     assert hasattr(distributed, "_rapids_patched")
 
 
-@pytest.mark.parametrize("python_version", [(3, 11, 9), (3, 11, 8)])
-@run_test_in_subprocess
-def test_dask_accessor(python_version):
-    import sys
-
-    import dask
-
-    # Simulate the version of Python and Dask needed to trigger vendoring of the
-    # accessor module.
-    sys.version_info = python_version
-    dask.__version__ = "2023.4.1"
-
-    from dask.dataframe import accessor
-
-    assert (hasattr(accessor, "_rapids_vendored")) == (python_version >= (3, 11, 9))
```

Comment:
We shouldn't include this test anymore. The accessor fix was merged upstream a very long time ago, and so our "vendored" accessor file was very stale anyway (removed with this PR).